jupyterlab/labbox_ephys_widgets_jp/src/extensions/curation/CurationSortingView/ControlPanel.tsx

```diff
@@ -1,43 +1,44 @@
 import { Grid, Paper } from '@material-ui/core';
 import React, { FunctionComponent, useCallback } from 'react';
-import { SortingCuration, SortingCurationDispatch } from '../../extensionInterface';
+import { SortingCuration, SortingCurationDispatch, SortingSelection, SortingSelectionDispatch } from '../../extensionInterface';
 
 type Props = {
     width: number
     curation: SortingCuration
     curationDispatch: SortingCurationDispatch
-    selectedUnitIds: number[]
+    selection: SortingSelection
```

Comment:
replacing selectedUnitIds and onSelectedUnitIdsChanged by `selection` and `selectionDispatch` throughout